relnotes/verify.feature.md

```diff
@@ -0,0 +1,9 @@
+* `ocean.core.Verify`
+
+  New module with a single function, `verify`, intended to serve as drop-in
+  replacement for `assert` to comply to [Sociomantic assert/enforce
+  policies](https://github.com/sociomantic-tsunami/sociomantic/blob/master/Code/assert_vs_enforce.md)
```

Comment:
@mihails-strasuns-sociomantic @gavin-norman-sociomantic normally I would take more time to discuss the structure but I trust your judgement and simply don't have the bandwidth right now. Please do propose a language-based structure if need be.